Spt 34 menu item wrapping

charleskoehl opened the pull request:
## What this PR does
- Fixes word-wrapping of menu items by changing overflow to hidden, which prevents scroll-bar appearing within menu. This issue was only reproduced in Chrome 51, Windows 8, 1366 x 768 within browserstack.com
## How should this be manually tested?
- Pull branch SPT-34.
- npm link to IHv2
- Test for word-wrapping and scroll-bar in Windows 8, Chrome 51, display resolution 1366 x 768.
## What are the relevant tickets?
- SPT-34

Commit:
code indentation

Commit:
prevent menu items from growing in height when text wraps (reproduced in Chrome 51, Windows 8, 1366 x 768)

uniqname commented on index.js:
```diff
@@ -38,30 +38,31 @@ export default class Drawer extends Component {
       transition: '.25s max-height linear',
       maxHeight,
-      overflow: 'auto'
+      overflow: 'hidden'
```
The only issue I see with this (and the reason it was `auto` to begin with) is if the menu contents extends beyond the height of the window (e.g. a phone in landscape mode).

charleskoehl replied:
Then perhaps overflow-x:hidden would be better.

Commit:
Change test-height of drawer to '9999px' because '100vh' may be less than height of menu when viewport is very short, as in iPhone landscape.
Keep overflow:hidden because 'overflowX: hidden; overflowY: auto' didn't do anything.
